subsys/nrf_security/Kconfig.psa

```diff
@@ -70,6 +70,23 @@ config MBEDTLS_PSA_KEY_SLOT_COUNT
 	  transparent keys that we can have open concurrently.
 	  The default value comes from MBedTLS.
 
+config MBEDTLS_PSA_STATIC_KEY_SLOTS
+	bool "Enable static key slots for PSA"
+	help
+	  When this option is enabled the PSA core doesn't use the heap to store keys.
+	  It uses a static buffer to hold the key material.
+
+config MBEDTLS_PSA_KEY_STORE_DYNAMIC
+	bool
+
+config MBEDTLS_PSA_STATIC_KEY_SLOT_BUFFER_SIZE
+	int "PSA static keyslot buffer size in bytes"
+	depends on MBEDTLS_PSA_STATIC_KEY_SLOTS
+	default 65
```

Comment:
Can this be based on enabled key-types instead of just picking one ECC public key as a basis?

Reply:
But I will write more in the help text to explain the 65.